Makefile

```diff
@@ -172,12 +176,18 @@ testunit: libpodimage
 localunit: varlink_generate
```

Comment:
OK. Just thought maybe we should not failed the CI if the test utils have some problem. But it make sense to run this in CI to make sure the integration tests are valid.

Reply:
Yes for sure.  At a minimum, maybe someone makes a small typo that breaks test utils, we want to catch that problem early rather than waiting for system-tests to run (and fail) much later.